src/Entities/Entity.cpp

```diff
@@ -946,35 +946,24 @@ void cEntity::HandlePhysics(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 	GET_AND_VERIFY_CURRENT_CHUNK(NextChunk, BlockX, BlockZ);
 
 	// TODO Add collision detection with entities.
-	auto DtSec = std::chrono::duration_cast<std::chrono::duration<double>>(a_Dt);
-	Vector3d NextPos = Vector3d(GetPosX(), GetPosY(), GetPosZ());
+	auto DtSec = std::chrono::duration_cast<std::chrono::duration<double>>(a_Dt).count();
```

Comment:
Since this is used in several places, it could do with being an inline function, probably.